src/integrationTest/resources/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
+services:
+  azure-storage:
+    image: arafato/azurite:2.6.5
```

Comment:
what is this for?

Reply:
That is for azure storage docker which is not needed for notification service